Doc/library/os.rst

```diff
@@ -3269,8 +3270,8 @@ features:
    .. versionchanged:: 3.6
       Accepts a :term:`path-like object`.
 
-   .. versionadded:: 3.7
-      Added :attr:`f_fsid`.
+   .. versionchanged:: 3.7
+      Added the :attr:`f_fsid` attribute.
```

Comment:
Note to self: we need to document this attribute (in another PR).

Reply:
Yep, noticed that; there's also a bunch of others defined alongside it that should also presumably be documented, so I didn't suggest it here.